packages/react-server/core/__tests__/integration/internalServerError/pages/InternalServerErrorException.js

```diff
@@ -3,8 +3,9 @@ import React from "react"
 export default class InternalServerErrorException {
 	handleRoute() {
 
-		// Not throwing an `Error` since this is easier to match.
-		throw "died";
+		// Need to throw a real error here, since CLS is going to
+		// assign a property to the error to track its source context.
+		throw new Error("died");
```

Comment:
Yeah, it gets no context and the `logger.error()` message is about failing to bind the context instead of about failing to navigate.  The message includes the thrown text either way.

This diff is mostly just about not leaving examples of anti-patterns laying around in tests. :sweat_smile: